src/test/java/seedu/address/logic/parser/transaction/EditTransactionCommandParserTest.java

```diff
@@ -0,0 +1,223 @@
+package seedu.address.logic.parser.transaction;
+
+import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.commands.CommandTestUtil.AMOUNT_DESC_INTERN;
```

Comment:
minor nit, would it be better if we replaced all the static imports with just importing the CommandTestUtil class? would make the code look cleaner

Reply:
Thanks for the suggestion!